firehose_test.go

```diff
@@ -696,6 +723,12 @@ outer:
 	cancel()
 }
 
+type indexFile struct {
```

Comment:
is it possible to use this instead of redefineing? https://github.com/cloudchacho/hedwig-firehose/pull/26/files#diff-a9fa94e3be3471ee03f13dc64fb6319fff832ae4ffcee07fe6dd8767f0af48afL82